t/compile.t

```diff
@@ -18,11 +18,8 @@ map {
     }, catfile($thisdir, $_));
 } qw(../bin ../lib);
 
-for (@tests) {
-    my $output = `perl -I "$libpath" -c $_ 2>&1`;
-    my $ok = ($? >> 8 == 0);
-    print $output unless $ok;
-    ok($ok, "$_ syntax check");
+foreach my $file (@tests) {
+    require_ok $file;
```

Comment:
I'm backing out the commit for requires_ok entirely.  I looked at it further and Test::Compile doesn't give me the output I was expecting, and the current behavior of t/compile.t is enough for me.